packages/evo-sdk-common/src/evo/common/utils/version.py

```diff
@@ -50,8 +50,8 @@ def get_package_details(candidate: str) -> PackageDetails:
             candidate, *_ = candidate.rpartition(".")
         else:
             return PackageDetails(
-                name=package_metadata["name"],
-                version=package_metadata["version"],
+                name=package_metadata["name"] if "name" in package_metadata else "unknown",
+                version=package_metadata["version"] if "version" in package_metadata else "unknown",
```

Comment:
`Mapping.get(...)` is probably cleaner, IMO

```suggestion
                name=package_metadata.get("name", "unknown"),
                version=package_metadata.get("version", "unknown"),
```

Reply:
Also, because this is provided by `evo-sdk-common`, do we want to default to the package name `evo-sdk-common`? (similar to the other fallback)

Reply:
Ah yes, @GriffinBaxterSeequent pointed that out as well. I have updated this including using the `evo-sdk-common` and version of `0.0.0` as suggested by Griffin also.